__tests__/app.test.js

```diff
@@ -0,0 +1,36 @@
+const request = require("supertest");
+const seed= require("../db/seeds/seed");
+const data = require("../db/data/test-data/index");
+const connection = require("../db/connection");
+const app = require("../app");
+
+beforeEach(() => {
+    return seed(data)
+});
+
+afterAll(() => {
+    return connection.end()
+});
+
+describe.only("GET /api/topics", () => {
```

Comment:
.only shouldn't remain in committed code, it shows you haven't ran the tests fully before committing!

Reply:
I'll let you off though because you only have one describe block so far